code/script.js

```diff
@@ -0,0 +1,95 @@
+
+//-----DOM-----//
+const profileInfo = document.getElementById("profile-info")
+const projectsContainer = document.getElementById("projects")
+
+
+//-----Global variables -----//
+const userName = 'toal13'
+const authorName = 'Tomoyo Alvåg'
+const API_URL = `https://api.github.com/users/${userName}/repos`;
+const USER_API = `https://api.github.com/users/${userName}`;
+
+
+//-----Token-----//
+const options = {
+    method: 'GET', //POST, PATCH, DELETE
+    headers: {
+        Authorization: `token ${TOKEN}`
+    }
+}
+
+
+//------Profile-----//
+const getUser = () => {
+    fetch(USER_API, options)
+        .then(res => res.json())
+        .then((data) => {
+            profileInfo.innerHTML += `
+            <h2><img src = "./github.png"><a href="https://github.com/${userName}" target="_blank">${data.login}</a></h2>
```

Comment:
I love the small GitHub img next to your name! And it looks great in desktop mode that you placed this section and the chart to the left of the projects!